src/main/frontend/components/block.cljs

```diff
@@ -290,8 +290,7 @@
      (if (string/ends-with? s ".excalidraw")
        [:a.page-ref
         {:href (rfe/href :draw nil {:file (string/replace s (str config/default-draw-directory "/") "")})
-         :on-click (fn [e]
-                     (util/stop e))}
+         :on-click (fn [e] (util/stop e))}
```

Comment:
Change it to `preventDefault`?

Reply:
It seems that just change it to `preventDefault` doesn't work well, I removed the `:href` part and set it to redirect to the draw page when clicking on the link.